### ISSUE TITLE
Finetuning on Tagging task for custom dataset outputs f1 = 0 and 

### ISSUE THREAD

**Issue by etetteh** [google-research/electra#110]
I created the following class
```
class BC5C(TaggingTask):
  """Entity-level text chunking."""

  def __init__(self, config, tokenizer):
    super(BC5C, self).__init__(config, "bc5c", tokenizer, True)
```
The format of the data can be inferred [here](https://github.com/google-research/electra/files/5654897/test.txt).

The training during finetuning works okay, but the evaluation on the dev set outputs zero (0) for all metrics except the loss, which is often 0.4.

Has anyone encountered this issue before, and what is the fix?

**Comment by Joseph-Vineland** [comment]
Did you find a solution?  If so, can you please share your code & what you did?

I also have a custom seqeunce tagging task, but ELECTRA is failing just before the final metrics are reported.

EDIT: I used your code snippet and file to find the solution.  Thanks!
